app/tests/utils/dataFormatting/formatPhone.test.ts

```diff
@@ -0,0 +1,21 @@
+import { formatPhone } from '@utils/dataFormatting'
+
+it('should return a formatted phone number', () => {
+  const result = formatPhone('1231231234')
+  expect(result).toBe('123-123-1234')
+})
+
+it('should strip non digits', () => {
+  const result = formatPhone('123abc123abc1234')
+  expect(result).toBe('123-123-1234')
+})
+
+it('should return empty string if the phone number is too long', () => {
```

Comment:
nit: alternatively could let it remain unformatted and let the validation error happen? As a user, I think I'd be annoyed/confused to see my phone number disappear.

Reply:
@sawyerh Interesting! Thank you for the feedback!